### PR TITLE
Initial td-migration document and test scripts

### PR DESCRIPTION
**Pull request by LeiZhou-97**
1. Add the td-migration doc for customers who could follow the single or cross host td-migration test steps to verify its basic function
2. Develop some td-migration required test scripts to simplify its usage.

Signed-off-by: Zhou, Lei <lei.zhou@intel.com>

### CODE REVIEW

**Comment by kenplusplus** [comment]
@ruomengh  @dongx1x  how about put the tools to utils directory?

File: td-migration/README.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+# create the source user TD on source platform
+sudo ./sourceTD.sh -i /path/to/image -b grub
+
+# create the destination user TD on destination platform
+sudo ./destTD.sh -i /path/to/image -b grub
+```
+
+### 3.3 Start Pre-Migration
+
+This step has a little different with the single host live migration.
```
Comment (ruomengh) [inline td-migration/README.md:136]: Typo: different -> difference

File: td-migration/README.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
+# TD Migration
+
+## 1. Overview
+This feature is to meet the CSP who may want to relocate/migrate an executing Trust Domain (TD) from a source Trust Domain Extension (TDX) platform to a destination TDX platform in the cloud environment. Currently, td-migration only works on TDX-Module 1.5, TDX-Module 1.0 do not support this feature.
```
Comment (ruomengh) [inline td-migration/README.md:4]: It's better to be clear on which version/tag should be used for TDX 1.5 - the kernel version  (v6.2) and corresponding tag (2023ww15)

**Review by ruomengh — CHANGES_REQUESTED** [review]

**Comment by LeiZhou-97** [comment]
> @ruomengh @dongx1x how about put the tools to utils directory?

Updated, "td-migration" has move to utils dir.

**Comment by LeiZhou-97** [comment]
@ruomengh  README has updated based on your advice. Pls. review it again.

**Review by ruomengh — APPROVED** [review]
LGTM

**Pull request merged**